utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -653,60 +665,154 @@ int main(int argc, char **argv)
-                        rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut16f, roiTensorPtrSrc, roiTypeSrc, handle);
-                    else if (inputBitDepth == 4)
-                        rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut32f, roiTensorPtrSrc, roiTypeSrc, handle);
-                    else if (inputBitDepth == 5)
-                        rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut8s, roiTensorPtrSrc, roiTypeSrc, handle);
-                    else
-                        missingFuncFlag = 1;
+                startWallTime = omp_get_wtime();
+                if (inputBitDepth == 0 || inputBitDepth == 1 || inputBitDepth == 2 || inputBitDepth == 5)
+                    rppt_crop_and_patch_gpu(d_input, d_input_second, srcDescPtr, d_output, dstDescPtr, roiTensorPtrSrc, cropRoi, patchRoi, roiTypeSrc, handle);
+                else
+                    missingFuncFlag = 1;
+
+                break;
+            }
+            case 34:
+            {
+                testCaseName = "lut";
+
+                Rpp32f *lutBuffer;
+                CHECK(hipHostMalloc(&lutBuffer, 65536 * sizeof(Rpp32f)));
+                CHECK(hipMemset(lutBuffer, 0, 65536 * sizeof(Rpp32f)));
+                Rpp8u *lut8u = reinterpret_cast<Rpp8u *>(lutBuffer);
+                Rpp16f *lut16f = reinterpret_cast<Rpp16f *>(lutBuffer);
+                Rpp32f *lut32f = reinterpret_cast<Rpp32f *>(lutBuffer);
+                Rpp8s *lut8s = reinterpret_cast<Rpp8s *>(lutBuffer);
+                if (inputBitDepth == 0)
+                    for (j = 0; j < 256; j++)
+                        lut8u[j] = (Rpp8u)(255 - j);
+                else if (inputBitDepth == 3)
+                    for (j = 0; j < 256; j++)
+                        lut16f[j] = (Rpp16f)((255 - j) * ONE_OVER_255);
+                else if (inputBitDepth == 4)
+                    for (j = 0; j < 256; j++)
+                        lut32f[j] = (Rpp32f)((255 - j) * ONE_OVER_255);
+                else if (inputBitDepth == 5)
+                    for (j = 0; j < 256; j++)
+                        lut8s[j] = (Rpp8s)(255 - j - 128);
+
+                startWallTime = omp_get_wtime();
+                if (inputBitDepth == 0)
+                    rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut8u, roiTensorPtrSrc, roiTypeSrc, handle);
+                else if (inputBitDepth == 3)
+                    rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut16f, roiTensorPtrSrc, roiTypeSrc, handle);
+                else if (inputBitDepth == 4)
+                    rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut32f, roiTensorPtrSrc, roiTypeSrc, handle);
+                else if (inputBitDepth == 5)
+                    rppt_lut_gpu(d_input, srcDescPtr, d_output, dstDescPtr, lut8s, roiTensorPtrSrc, roiTypeSrc, handle);
+                else
+                    missingFuncFlag = 1;
+
+                CHECK(hipHostFree(lutBuffer));
```

Comment:
There's some issue in the merge, please always recheck. Lines are repeated. There will be a doubleFree issue.